.github/workflows/tool-tests.yml

```diff
@@ -25,14 +29,10 @@ jobs:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v2
 
-      # Runs a single command using the runners shell
-      - name: Run a one-line script
-        run: echo Hello, world!
-
-      # Runs a set of commands using the runners shell
-      - name: Run a multi-line script
-        run: |
-          ls -la
-          export GH_TOKEN=$(GH_TOKEN)
-          export BASE_PATH=$(pwd)
-          python -m unittest ./python/tests/test_update_package_properties.py
+      - name: install requirements
+        run: python -m pip install -r python/requirements.txt
+      - name: Execute Unit tests
+        run: python -m unittest $(pwd)/python/tests/test_update_package_properties.py
```

Comment:
Do we need `$(pwd)/` here?

Reply:
Is there a reason we have 3 extra lines at the bottom?

Reply:
Removed both of them